docs/source/pages/howToWorkWithPlatform/adminApplication/users/users.rst

```diff
@@ -2,173 +2,191 @@
 
-3. Fill required fields on the user creation form, such as: first, last names, first part of email and select domain part. Also you can choose to create or not personal mailbox for this user
+===============
+Corporate users
+===============
 
-.. image:: pic_users/createUserForm.png
-   :width: 600
-   :align: center
+Corporate user is a special type of user that can be created, menaged and removed by an administrator of an accoun with at least one active domain. Such users can only be created with an email address from a verified domain of the account. All users using such emails during self-registration will also be automatically assigned to the this account. More info on the domains management can be found :ref:`here <domains>`.
 
-4. Click the "Create" button
-5. After this steps user will be created and for admin opens user permissions configuration page. How to add specific permissions to the mailbox and/or account you can find :ref:`Add permissions to account <accountPermissions>` and :ref:`Add permissions to mailbox <mailboxPermissions>`
-6. As soon as permissions defined, click the "Save button" in the header of the page and user permissions will be applied
+Users assignment to a domain account
+====================================
 
-.. _howToAuth:
+As soon as you've added, verified and activated a domain you can create and manage users with it.
 
-How to authorized as a corporate user
-=====================================
+* Users registered before a domain activation will be automatically assigned to the domain account after the domain activation. They will not lose any previosly owned permissions to other accounts and mailboxes.
+* Users registered after a domains activation will be automatically assigned to the domain account immediately. They will not have any account and mailbox permissions until granted by the account administrator. Personal mailbox with a specified role may or may not be automatically created for such users based on the :ref:`account configuration <accountSettings>`.
 
-1. Open email "You are registered on the electronic document management platform"
+In case of a domain unverification or deactivation all domain users will retain all assigned account and mailbox permissions, but creation or self-registration of new users in this domain be disabled. In case of an account deletion all user will lose acces to the account and its' mailboxes, but retain a corporate user status, meaning that they will not be able to create accounts and delete their user profiles.
```

Comment:
an account deletion all user will lose acces to

an account deletion all user will lose access to